projects/extension/src/utils/debug.test.ts

```diff
@@ -0,0 +1,25 @@
+import { debug } from './debug';
+import { jest } from '@jest/globals';
+
+beforeEach(() => {
+  jest.resetModules();  
+  process.env = { NODE_ENV: 'development' };
```

Comment:
Should probably reset the environment after the tests to avoid surprises in other tests that depend on it